test/test_torch.py

```diff
@@ -3645,6 +3645,26 @@ def test_normal_kstest(self, device, dtype):
                 res = stats.kstest(t.cpu().to(torch.double), 'norm', args=(mean, std))
                 self.assertTrue(res.statistic < 0.1)
 
+    # Ensure that normal raises appropriate error when `std` <= 0
+    def test_normal_std_error(self, device):
```

Comment:
Move this to test_tensor_creation_ops.py:

https://github.com/pytorch/pytorch/blob/316f0b89c3aa51329f40a13c00de587da60faa66/test/test_tensor_creation_ops.py#L2807